python_scripts/shellies_discovery.py

```diff
@@ -1519,6 +1521,12 @@
     DEVICE_CLASS_WINDOW,
 ]
 
+OPTIONS_SENSOR_RETURNING = {
+    KEY_ENABLED_BY_DEFAULT: True,
```

Comment:
This sensor will not be needed by all users, so let it be disabled by default.
```suggestion
    KEY_ENABLED_BY_DEFAULT: False,
```